src/main/java/buildcraftAdditions/items/ItemBase.java

```diff
@@ -19,25 +14,11 @@
  */
 
 public class ItemBase extends Item {
-	public IIcon icon;
-	public String name;
```

Comment:
i don't think this can be removed like that, pretty sure it's used in items extending itemBase or otherwise i've put it there for future use